src/content/docs/en/technology/chain/differences.mdx

```diff
@@ -39,7 +38,6 @@ Disabled EIPs until the London fork:
 
 - [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559): Fee market change and EIP-1559 transaction type
 - [EIP-2930](https://eips.ethereum.org/EIPS/eip-2930): Optional access lists
```

Comment:
These two are supported now (though we could mention that the EIP-1559 fee market behaves differently: the sequencer sets the L2 base fee (based on the L1 base fee), and we don't burn ETH.

Reply:
thanks, updated!